### DISCUSSION TITLE
Hacktoberfest 2022

### DISCUSSION THREAD

**Discussion by Preet-Sojitra** [formbricks/formbricks#88] [General Requests] [answered]
Is this repository participating in Hacktoberfest

**Comment by mattinannt** [comment, marked as answer]
Hi @Preet-Sojitra , just checked out Hacktoberfest; sounds pretty great. I added the `hacktoberfest` topic to the repository and will now add some issues and feature requests and label them accordingly. So feel free to participate 🤗
If you have other features / bugs / .. you would like to work on please do so 💪
Ping me if you need any help. best way for that is our discord: https://snoopforms.com/discord
